### PR TITLE
unify benchmarks

### LINKED ISSUE #273: DOC 3D benchmarks
Currently none are listed in the user's guide. We should probably include something here.

### LINKED ISSUE #359: Normalize benchmarks
Right now, the different `compute_speed.py` scripts have very different running times. We should normalize these so that they take about the same time. Of course, this has to be balanced with ensuring large enough batch sizes to ensure maximum efficiency.

### PR DESCRIPTION
**Pull request by lostanlen**
Closes #359

### CODE REVIEW

File: benchmarks/benchmarks/common.py [hunk 1]
```diff
@@ -1,2 +1,6 @@
 import torch
 torch.manual_seed(0)
+
+# This constant corresponds to the total number of samples per batch. It is
+# shared across all benchmarks of scattering: 1D, 2D, and 3D.
+SCATTERING_BENCHMARK_SIZE = 2**21
```
Comment (janden) [inline benchmarks/benchmarks/common.py]: Spaces around `**`.
